app/controllers/donations_controller.rb

```diff
@@ -10,13 +10,27 @@ def create
     end
   end
 
+  def update
+    @donation = find_donation(params[:id])
+    if @donation.update(donation_params)
+      redirect_to controller: "donors", action: "show"
```

Comment:
I don't think this would work, because you can't redirect to a `show` page without providing an id of the record to `show`. 😀 

Normally we'll just pass a path or URL to `redirect`. We should also provide some feedback to the user whether the update was successful or not. 😀 

```
if @donation.update(donation_params)
  redirect_to donor_path(@donation.donor), notice: "Donation updated successfully."
else
  redirect_to donor_path(@donation.donor), error: "Couldn't update donation."
end
```